scripts/gobuster.sh

```diff
@@ -5,5 +5,9 @@ git clone https://github.com/oj/gobuster /opt/gobuster
 
 cd /opt/gobuster && \
   make && \
-  cp /opt/gobuster/gobuster /usr/local/bin/gobuster
-
+  cp /opt/gobuster/gobuster /usr/local/bin/gobuster && \
+  git clone --no-checkout https://github.com/v0re/dirb.git && \
```

Comment:
Make sure we're back in the proper directory:

```suggestion
  cd /opt && \
  git clone --no-checkout https://github.com/v0re/dirb.git && \
```